docs/config.json

```diff
@@ -31,6 +31,10 @@
         {
           "title": "Changelog",
           "slug": "/changelog/"
+        },
+        {
+          "title": "Teleport Assist",
+          "slug": "/ai-assist/"
         }
```

Comment:
Is this where we want this? Not as a top-level section? @klizhentas

Reply:
I would like to make an exception for this feature, I would like it to be visible. After a while, we will move it from Home